lib/Runtime/Types/EdgeJavascriptTypeId.h

```diff
@@ -120,6 +120,7 @@ enum TypeId
     TypeIds_ActivationObject = 73,
     TypeIds_SpreadArgument = 74,
     TypeIds_ModuleNamespace = 75,
+    TypeIds_ListIterator = 76,
```

Comment:
>TypeIds_ListIterator [](start = 4, length = 20)

Should we group this with the other iterators? Seems like the ListIterator is a built-in dynamic object?